### PR TITLE
contrib/brightnessctl: new package (0.5.1)

### PR DESCRIPTION
**Pull request by Froggo8311**

### CODE REVIEW

File: contrib/brightnessctl/template.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+pkgver = "0.5.1"
+pkgrel = 0
+build_style = "makefile"
+make_cmd = "gmake"
+make_install_args = ["UDEVDIR=/usr/lib/udev/rules.d"]
+make_use_env = True
+hostmakedepends = [ "gmake" ]
+pkgdesc = "Program to read and control device brightness"
+maintainer = "Froggo <froggo8311@proton.me>"
+license = "MIT"
```
Comment (nekopsykose) [inline contrib/brightnessctl/template.py:11]:
```suggestion
license = "X11"
```

**Comment by Froggo8311** [comment]
Applied patch & rebased & squashed

**Comment by q66** [comment]
the license is MIT, not X11

the commit message should follow the expected format `category/name: new package (version)`

**Comment by q66** [comment]
otherwise looks fine, so just fix that

**Comment by Froggo8311** [comment]
@q66 lmfao @nekopsykose trolled me ig. i had it as mit but she changed it to x11 :skull: 

give me a moment

**Comment by q66** [comment]
they are the same license, but X11 has the following added:

```
Except as contained in this notice, the name of the X Consortium shall not be used in advertising or otherwise to promote the sale, use or other dealings in this Software without prior written authorization from the X Consortium.

X Window System is a trademark of X Consortium, Inc.
```

**Comment by q66** [comment]
you haven't fixed the commit message

**Comment by Froggo8311** [comment]
> they are the same license

yeah i know lmao


> you haven't fixed the commit message

wtf okay

**Comment by q66** [comment]
also nevermind, this *is* the X11 license

i was looking at a tag which had it as MIT and it got changed later?

**Comment by Froggo8311** [comment]
smh okay give me a moment to fix again

**Pull request closed without merging**